Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "hyper"
-version = "0.12.32" # don't forget to update html_root_url
+version = "0.13.0-a.0" # don't forget to update html_root_url
```

Comment:
do you prefer `a` over `alpha`?

Reply:
I dunno, I just put it there so that version is considered different, I didn't expect to release it yet... :shrug: